app/controllers/users/registrations_controller.rb

```diff
@@ -5,7 +5,7 @@ class Users::RegistrationsController < Devise::RegistrationsController
   after_action :allow_iframe
 
   layout "application"
-  layout "application_narrow", only: %i[new edit pending]
+  layout "application_narrow", only: %i[new create edit pending]
```

Comment:
est-ce qu'il faut aussi ajouter le update vu qu'il y a le edit ? Ou alors est-ce que la distinction entre le narrow et le normal devrait plutôt être au niveau de la vue plutôt que du layout ?

Reply:
bien vu victor il y a en effet un problème de saut de layout similaire lorsqu’il y a une erreur de validation sur le formulaire Votre compte 

<img width="1258" alt="image" src="https://github.com/user-attachments/assets/8ce67352-52e5-4f4a-bfa4-73d719b90e69">

(oui j’ai RDVSP en dark mode forcé 🌔 )

On peut effectivement rajouter update pour corriger le pb ici

```suggestion
  layout "application_narrow", only: %i[new create update edit pending]
```